### DISCUSSION TITLE
i can't start my project

### DISCUSSION THREAD

**Discussion by denizyoldas** [react-page/react-page#1194] [Q&A] [answered]
i read the [Getting started](https://react-page.github.io/docs/#/quick-start?id=getting-started)
but not working project
![Screen Shot 2022-06-02 at 11 36 34](https://user-images.githubusercontent.com/36186348/171590481-ff0abdee-7a47-4b1b-9f3b-4e13191df163.png)
![Screen Shot 2022-06-02 at 11 36 51](https://user-images.githubusercontent.com/36186348/171590491-392f96a8-29c0-41af-95dd-0325ab34662e.png)

**Comment by macrozone** [comment, marked as answer]
hi @denizyoldas, use react 17. react 18 is currently not supported by all dependencies of react-page.

follow this milestone for react 18 support: https://github.com/react-page/react-page/milestone/14
